tensorboard/plugins/hparams/tf_hparams_main/BUILD

```diff
@@ -25,14 +27,13 @@ tf_web_library(
 tensorboard_webcomponent_library(
     name = "legacy",
     srcs = [":tf_hparams_main"],
-    visibility = [":tf_hparams_main_users"],
+    visibility = ["//tensorboard/plugins/hparams/external_users:whitelist"],
```

Comment:
This does not appear to exist. Can you double check please?

Reply:
I am not so thrilled about having a ghost target like this even if it not used in GitHub. Can we do something about it? (Perhaps a shim target that gets replaced internally?)